### ISSUE TITLE
User-Subscription

### ISSUE THREAD

**Issue by Carl-Granstrom** [weleoka/A02_u01#2]
Jag vet inte exakt hur du har tänkt dig strukturen, men nog bör Subscription vara kopplad mot User istället för main? Anropet går från main, men Subscription-objekten hör till användare. Om man vill lagra flera subs kan man ha dom i en Subscription[]. Men bäst är kanske att ha en sub med metoder för att förlänga den osv. Tror jag ska fundera mer på min egen sub-mekanism och försöka få den att funka så. Orkade bara inte med datumhanteringen ursprungligen.

**Comment by weleoka** [comment]
Agreed. Jag har lirkat lite fram och tillbaks med detta. Nytt UML uppe nu. Dock är subsc fortfarande på WakeUp facade.

Jag ser fram emot vad lärarna kommer att säga om det sedan. Hela UML grejen är ju överkurs men jag tycker det är kul. Bra mycket bättre än et sudoku iaf!

On December 1, 2018 3:01:12 PM GMT+01:00, "Carl Granström" <notifications@github.com> wrote:
>Jag vet inte exakt hur du har tänkt dig strukturen, men nog bör
>Subscription vara kopplad mot User istället för main? Anropet går från
>main, men Subscription-objekten hör till användare. Om man vill lagra
>flera subs kan man ha dom i en Subscription[]. Men bäst är kanske att
>ha en sub med metoder för att förlänga den osv. Tror jag ska fundera
>mer på min egen sub-mekanism och försöka få den att funka så. Orkade
>bara inte med datumhanteringen ursprungligen.>
>>
>-- >
>You are receiving this because you are subscribed to this thread.>
>Reply to this email directly or view it on GitHub:>
>https://github.com/weleoka/A02_u01/issues/2

--
sent with k9email client for mobile.
